examples/predictors/lstm/xprize_predictor.py

```diff
@@ -74,7 +74,7 @@ class XPrizePredictor(object):
     A class that computes a fitness for Prescriptor candidates.
     """
 
-    def __init__(self, path_to_model_weights, data_url, cutoff_date_str):
```

Comment:
Removed `cutoff_date_str` argument from constructor, and related private methods